packages/core/src/plugins/popup-controller.class.ts

```diff
@@ -29,16 +40,48 @@ export class PopupController {
       throw new Error('Please allow popup windows');
     }
 
-    pluginWindow.moveTo(0, 0);
-    pluginWindow.resizeTo(screen.width, screen.height);
+    const sizeAndPosition = this.getWindowSizeAndPositionFromStorage();
+    pluginWindow.moveTo(sizeAndPosition.position.x, sizeAndPosition.position.y);
+    pluginWindow.resizeTo(sizeAndPosition.size.width, sizeAndPosition.size.height);
 
     window.addEventListener('unload', this.onUnload);
 
     return pluginWindow;
   }
 
+  private getWindowSizeAndPositionFromStorage(): WindowSizeAndPosition {
+    const defaultSizeAndPosition = {
+      position: { x: 0, y: 0 },
+      size: { width: screen.width, height: screen.height },
```

Comment:
Wondering if the default might be better as a fraction of the full screen to start?

Reply:
Yeah, might be. Or possibly we could skip moving it and resizing it if there is no previous size? Since then the browser will automatically use the previous window size (meaning the "parent" size)?